tests/unit/services/tokens.unitTest.ts

```diff
@@ -0,0 +1,38 @@
+import {getValidJwt} from "../../../src/services/tokens";
+
+describe('getValidJwt()', () => {
+  it('should fail on blank authorization token', async () => {
+    expect(() => { getValidJwt('') }).toThrowError('no caller-supplied-token');
+  });
+
+  it('should fail on non-Bearer authorization token', async () => {
+    expect(() => { getValidJwt('not a Bearer') }).toThrowError('must start with \'Bearer \'');
```

Comment:
I wonder if we want to store those error messages we share across test files and service in a file so if we update the constant and we reference them we have to do it in one place in case we get asked to do it in the future?

Reply:
Some of these error messages are templated, so it would make it rather annoying to manage for the whole project. As agreed on the call - no action